src/licenses/entities/license/licenseFindResult.ts

```diff
@@ -164,6 +174,9 @@ export type LicenceFindDataKeywords = {
   [LicenseFindResultFields.COLUMN_RESELLER_NAME]?: DataKeywords;
   [LicenseFindResultFields.COLUMN_RESELLER_REF]?: DataKeywords;
   [LicenseFindResultFields.COLUMN_SEAT]?: DataKeywords;
+  [LicenseFindResultFields.COLUMN_SECURITY]?:
+    | SecurityFindResultDataKeywords
+    | undefined;
```

Comment:
You can just do:  ` [LicenseFindResultFields.COLUMN_SECURITY]?: SecurityFindResultDataKeywords`  the '?' indicates that the parameter is optional (could be undefined)